src/averageAges.js

```diff
@@ -14,12 +20,21 @@
  *
  * @return {number}
  */
+const MALE_SEX = 'm';
+const FEMALE_SEX = 'f';
+const CUNTURY_COEFFICIENT = 100;
+
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const mensOfCenturyArray = people
+    .filter(person => person.sex === MALE_SEX)
+    .filter(person => century
+      ? century === Math.ceil(person.died / CUNTURY_COEFFICIENT) : true);
```

Comment:
You can use 1 filter instead, just move checks from the second filter to the first.

```suggestion
    .filter(person => (
    person.sex === MALE_SEX && (century 
      ? Math.ceil(person.died / CUNTURY_COEFFICIENT) === century 
      : true)
    ))
```